stubs/pycurl/pycurl.pyi

```diff
@@ -6,9 +6,9 @@ version: str
 
 def global_init(option: int) -> None: ...
 def global_cleanup() -> None: ...
-def version_info() -> tuple[
-    int, str, int, str, int, str, int, str, tuple[str, ...], Incomplete | None, int, Incomplete | None
-]: ...
+def version_info() -> (
+    tuple[int, str, int, str, int, str, int, str, tuple[str, ...], Incomplete | None, int, Incomplete | None]
+): ...
```

Comment:
All the changes look to me like clear improvements, except for maaaaaaybe this one, which I could go either way on.